packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -4,15 +4,18 @@ import styled from "styled-components"
 import { color, media, space } from "../../helpers"
 import { CloseIcon } from "../../svgs"
 import { ArtsyLogoBlackIcon } from "../../svgs/ArtsyLogoBlackIcon"
+import { usePrevious } from "../../utils/usePrevious"
 import { Box } from "../Box"
 import { Flex } from "../Flex"
 import { Spacer } from "../Spacer"
 import { Serif } from "../Typography"
 
+/**
+ * refreshModalContentKey should change if the modal displays new content and should fade in/fade out with content update
```

Comment:
@damassi there's a comment here describing it.

Reply:
> refreshModalContentKey should change if the modal displays new content and should fade in/fade out with content update. If `refreshModalContentKey` does not change, the content updates immedately.

Reply:
Ah I see. Updating now.